### PR TITLE
update test-unit for insights-metrics

### PR DESCRIPTION
**Pull request by SherinV**
Unit test is failing here https://github.com/stolostron/insights-metrics/pull/32
Related Issue: stolostron/backlog#15572

### CODE REVIEW

File: ci-operator/config/stolostron/insights-metrics/stolostron-insights-metrics-release-2.3.yaml [hunk 1]
```diff
@@ -3,6 +3,10 @@ base_images:
     name: ubi-minimal
     namespace: ocp
     tag: "8"
+  open-cluster-management_builder_go1.17-linux:
+    name: builder
+    namespace: open-cluster-management
```
Comment (jlpadilla) [inline ci-operator/config/stolostron/insights-metrics/stolostron-insights-metrics-release-2.3.yaml:8]: This one isn't changing to `stolostron`? Same a few lines below.

Comment (SherinV) [reply ci-operator/config/stolostron/insights-metrics/stolostron-insights-metrics-release-2.3.yaml:8]: Checked with @schmidtd. This should get updated automatically once we make the org rename changes to our repo docker file.

Comment (jlpadilla) [reply ci-operator/config/stolostron/insights-metrics/stolostron-insights-metrics-release-2.3.yaml:8]: Thanks for checking.

**Pull request merged**